src/cli/load.js

```diff
@@ -88,6 +92,7 @@ function die(std, message) {
 export type LoadOptions = {|
   +output: RepoId,
   +repoIds: $ReadOnlyArray<RepoId>,
+  +weightsPath?: string | null,
```

Comment:
if using `weightsPath?` then you don't need to do `string | null`, since the `?` already makes it optional.

Reply:
Changed it.